spec/features/manage_cmr/groups/filtering_groups_spec.rb

```diff
@@ -157,7 +157,7 @@
           end
 
           it 'displays the script tag in the user name' do
```

Comment:
Looks like we need to change the description on this test since it specifies displaying the script tag in the user name.